host/class/hid/usb_host_hid/idf_component.yml

```diff
@@ -4,7 +4,3 @@ description: USB Host HID driver
 url: https://github.com/espressif/esp-usb/tree/master/host/class/hid/usb_host_hid
 dependencies:
   idf: ">=4.4"
-targets:
```

Comment:
Why we removed targets from here?

Reply:
I put an explanation to the MR description:

As discussed, rather than adding new supported target into the idf_component.yml file, (linux in this case, and other targets in future) the target list was removed from (those class drivers buildable for linux) and a target compatibility will be handled during build.

So we don't have to update the `idf_component.yml` file for each new target.